src/components/Sidebar/Footer.vue

```diff
@@ -8,13 +8,15 @@
 
     <div v-on:click="$emit('closeMenu')">
       <router-link to="/contact" tag="li"
-        ><a class="contact icon" >Contact Us</a></router-link
+        ><a class="contact icon">Contact Us</a></router-link
```

Comment:
Is there a benefit to using this style of formatting that breaks the HTML tag into two lines? imho it's not very pretty!

If this is a means to stay under the max characters per row limit, could we increase that max? People have wider screens nowadays, editors can fit a bit more :P

Reply:
Agreed, it's bad. I'll see if I can fix this for #198.